impeller/renderer/backend/metal/formats_mtl.h

```diff
@@ -86,6 +92,8 @@ constexpr MTLPixelFormat ToMTLPixelFormat(PixelFormat format) {
       return SafeMTLPixelFormatBGR10_XR_sRGB();
     case PixelFormat::kB10G10R10XR:
       return SafeMTLPixelFormatBGR10_XR();
+    case PixelFormat::kB10G10R10A10XR:
+      return SafeMTLPixelFormatBGRA10_XR();
```

Comment:
Also add reverse conversion to `ToMTLBlendFactor` below.

Reply:
The reverse for this is in `FromMTLPixelFormat`.  I think that's what you were thinking, right?

Reply:
Ah yup, disregard